harness/tests/experiment/keras/test_tf_keras_trial.py

```diff
@@ -509,3 +514,75 @@ def make_workloads() -> workload.Stream:
     )
     with pytest.raises(ValueError, match="incompatible"):
         controller.run()
+
+
+@pytest.mark.tensorflow
+@pytest.mark.tf1_cpu
+def test_rng_restore(tmp_path: Path):
```

Comment:
if you are going to run multiple RNG tests inside the same process, can you set the global RNGs _wrong_ in between TrialControllers, so it's impossible to get a false-positive test?